packages/frontend-shared/vite.config.ts

```diff
@@ -53,6 +53,11 @@ const makePlugins = (plugins) => {
         cy: FileSystemIconLoader(path.resolve(__dirname, './src/assets/icons')),
         ...plugins.iconsOptions?.customCollections,
       },
+      iconCustomizer (collection, icon, props) {
+        if (icon.includes('_x16')) {
```

Comment:
At some point we could do this across the board for `_x24`, `_x32` and all the others - perhaps by checking the `_x${whatever}` and using that value for the inline height and width.